app/controllers/adoption_requests_controller.rb

```diff
@@ -20,6 +21,10 @@ def create
     end
   end
 
+  def show
+    @adoption_request = AdoptionRequest.find(params[:id]) 
+  end
```

Comment:
I don't think this is related to the feature you're adding. Maybe this sneaked in by accident.